zipline-gradle-plugin/src/main/kotlin/app/cash/zipline/gradle/ZiplinePlugin.kt

```diff
@@ -67,17 +67,21 @@ class ZiplinePlugin : KotlinCompilerPluginSupportPlugin {
     }
 
     target.tasks.withType(KotlinWebpack::class.java) { kotlinWebpack ->
```

Comment:
Unrelated to this change, but I'm pretty sure this causes all tasks to be configured. Gradle docs will have the lazy pattern. Separate PR for that...

Reply:
Follow-up coming up.

Reply:
https://github.com/cashapp/zipline/issues/1180